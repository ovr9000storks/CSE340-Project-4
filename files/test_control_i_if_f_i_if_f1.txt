
a , b , c, d ;
{

input c;
input d;
c = c + d;
d = c + d;
IF d > c {
	output a;
	output b;
	output c;
	output d;
	IF  a > c {
		a = a+1;
		b = b+1;
		output a;
		output b;
	}
	IF  c > a {
		a = a+2;
		b = b+3;
		output a;
		output b;
	}
}
a = 1  ;
b = 2 ;
IF b > a {
	output a;
	output b;
	output c;
	output d;
}
	
IF b > a {
        a = a+b;
	output a;
	IF a > b { 
		output a;
		b = a+b;
		IF b > a {
			output a;
			output b;
			IF a > b {
				output a;
				output b;
			}
		}
	}
	a = a+b;
	output a;
}
a = a*b;
output a;
output b;

input a;
input b;

IF a > b {
	output a;
	b = 1;
	a = a+b;
	output a;
	output b;
}

}
4 5 15 8 1 27 9
