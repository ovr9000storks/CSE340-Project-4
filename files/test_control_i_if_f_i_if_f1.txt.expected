0 0 9 14 2 3 1 2 9 14 3 3 3 5 8 40 5 15 16 1 